### PR TITLE
[QUIZ-496] Added quiz selected options field

### PR DESCRIPTION
**Pull request by lws803**
Add quiz selected options field to display it on results page.

image

Dependencies:

- [x] https://github.com/Constructor-io/quiz-service/pull/188
- [x] https://github.com/Constructor-io/quiz-service/pull/189

### CODE REVIEW

**Review by mocca102 — APPROVED** [review]
LGTM!

**Comment by lws803** [comment]
> LGTM!

Ooo @mocca102 would you mind helping me merge this? 😄 seems like I don't have perms for this repo

File: src/types/quizzes.d.ts [hunk 1]
```diff
@@ -83,6 +83,10 @@ export interface QuizResultsResponse extends Record<string, any> {
   quiz_version_id: string;
   quiz_session_id: string;
   quiz_id: string;
+  quiz_selected_options: Array<{
+    value: string;
+    has_attribute: boolean
```
Comment (esezen) [inline src/types/quizzes.d.ts:88]: Not important but does this need `;`?

**Review by esezen — CHANGES_REQUESTED** [review]
@lws803 The types are looking great but the type tests are failing since we don't have the new `quiz_selected_options` there. Do you mind updating `src/types/tests/quizzes.test-d.ts` ? You can run type tests with `test:types`

File: src/types/quizzes.d.ts (hunk 1 above)
Comment (lws803) [reply src/types/quizzes.d.ts]: Good catch! baee98ce951c3aed48d4f0a294ed574d561c7186

**Comment by lws803** [comment]
> @lws803 The types are looking great but the type tests are failing since we don't have the new `quiz_selected_options` there. Do you mind updating `src/types/tests/quizzes.test-d.ts` ? You can run type tests with `test:types`

Whoops, good catch! I've fixed those type issues. Realized I did not install npm packages for this repo yet

**Review by esezen — APPROVED** [review]
LGTM!

**Pull request merged**